src/main/java/com/getcloudcherry/survey/helper/RecordAnswer.java

```diff
@@ -126,14 +147,16 @@ public SurveyAnswers getAnswers() {
     public boolean checkIfPreFillExists(SurveyQuestions iQuestion) {
 
         for (String aTag : iQuestion.questionTags) {
```

Comment:
Just a heads up, We noticed that we are not displaying brand's logo anywhere (iSurveyResponse.logoURL). Please look into this.